docs/releasing.md

```diff
@@ -120,7 +120,13 @@ visible.
 
 [Prior art](https://github.com/metal3-io/ironic-image/pull/595)
 
-### Branches lifecycle
+### Update supported versions
+
+The supported versions matrix should be updated only later after CAPI release,
```

Comment:
s/CAPI/CAPM3/ ?

Bonus point for rephrasing the statement to highlight that you're talking about two different things. Or are you? I'm actually confused myself. Maybe structure it as "Do not update .... yet because it's only done after a CAPM3 release. Instead, update .... in ....."